README.md

```diff
@@ -57,9 +57,24 @@ Refer [concepts](doc/CONCEPTS.md) to understand the concepts referred to in this
   - Kafka on localhost:9092
   - ZooKeeper on localhost:2181
   - RabbitMQ on localhost:5672
-
 - Run tests: `make test`
 
+#### Running a cluster set up locally
+
+- `make setup-cluster` This starts
+    - 3 Kafka brokers on localhost:9091, localhost:9092 and localhost:9093
+    - Zookeeper on localhost:2181
+    - RabbitMQ on localhost:5672
+    
+- In case the cluster doesn't start up, one of the reasons could be
+invalid data in the volume location. Please delete the folder /tmp/ziggurat_kafka_cluster_data
+in such cases.
+    - rm -rf /tmp/ziggurat_kafka_cluster_data
```

Comment:
Can this be the first step in the `make setup-cluster` command?

Reply:
Done